### DISCUSSION TITLE
Possible usage with terminal PC?

### DISCUSSION THREAD

**Discussion by bakusiowywupek** [dz0ny/klipper-prusa-mk3s#35] [Q&A]
Hi, I have HP T630 terminal which I have been using before with Ender 3 V2 and now it's running OctoPrint for my Prusa. Now I'm curious can I use T630 for MK3S+? I have already RPI Pico which I have been using for ADXL for Ender. Maybe I can use it as an bridge between Rambo and PC?

**Comment by confuzled-dev** [comment]
This may be a discussion best suited for the Klipper discord server. 

https://discord.klipper3d.org/
